Vision/Vision.bs

```diff
@@ -54,19 +41,25 @@ Boilerplate: omit conformance
 		to represent the emergent consensus of most of our participants.
 	* Be timeless enough that it does not need frequent revision.
 
+# The Mission of W3C # {#mission}
+
+	The W3C leads the community 
+	in defining a World Wide Web that puts users first, 
+	by developing technical standards and guidelines 
+	to empower an ethical, equitable, informed, and interconnected society.
```

Comment:
```suggestion
	to empower an equitable, informed, and interconnected society.
```
I'm not sure what an "ethical" society is; I think our work definitely should be ethical, but I'm not sure how credible we can be in claiming our work helps make society "ethical".

Reply:
Hmm.  I think the baseline of principles - even having distinct principles of privacy, for example - have an effect in making our society more ethical.  I would be interested in more opinions here, though.

Reply:
Difference people hold different ethics. To pick a definition, which would be required if the word stays, is to marginalize others.

Reply:
I might edit this down to say something with an action word like "leading the Web..."  So: “developing technical standards for an ethical, equitable, informed and interconnected society for a Web that puts users first.”  Though I do prefer “for a Web that benefits humanity” vs. “puts users first”, I could be persuaded.